### ISSUE TITLE
Questions about Details of Applying DeepVariant

### ISSUE THREAD

**Issue by zjgbz** [google/deepvariant#320]
Hi DeepVariant Team,

I am exploring the potentiality of applying deepvariant in our project. And I have several questions that cannot be found in your paper. May I have your help?

1. Is your method also applicable to low-depth sequencing data or mainly applicable to high-depth sequencing data?

2. How could you organize your training data? To be specific, do you consider one variant of one chromosome of one individual as one tensor for training (one training case), and put all this kind of variants together as your training set to generate one trained model without considering if they are in the same chromosome or from one individual, rather than training model for each chromosome? Do I understand it correctly? If I understand it correctly -- one trained model for training variants across different chromosomes and individuals, what is the efficiency? Or how long will it take to generate one trained model and under what kind of computational resources, as well as how many variants are in your training set?

3. Do you publish any pre-trained model so that we can directly apply to our data to impute our variants?

I am not sure if it is proper to ask this kind of question here, so please feel free to delete it if here is not for this kind of question.

Thank you very much.

Best regards,
Min-Zhi

**Comment by sgoe1** [comment]
Hi Min-Zhi,

1. DeepVariant works well across a wide range of coverages. [This blog post](https://google.github.io/deepvariant/posts/2019-09-10-twenty-is-the-new-thirty-comparing-current-and-historical-wgs-accuracy-across-coverage/) provides some analysis on what performance looks like across different coverages.

2. There are a few questions here, so I'll refer you to 2 docs that I believe answer all of them:
	- [Blog post](https://google.github.io/deepvariant/posts/2020-02-20-looking-through-deepvariants-eyes/) 
	- [Training Case Study](https://github.com/google/deepvariant/blob/r0.10/docs/deepvariant-training-case-study.md)

	But to briefly address the individual questions:
	- How data is organized: [this doc](https://github.com/google/deepvariant/blob/r0.10/docs/deepvariant-details-training-data.md) outlines the data that we are using.
	- Your understanding is essentially correct, but see blog post linked above (and maybe [this one too](https://blog.dnanexus.com/2019-02-19-deep-dive-into-deepvariant/)) for detail.
	- The case study gives numbers on how long it takes to train a model with certain computational resources.

3. We provide [a few different ways](https://github.com/google/deepvariant#official-solutions) to use DeepVariant, and our recommended way is to use one of our pre-trained models through Docker. [This section](https://github.com/google/deepvariant/tree/r0.10/docs#quick-start-and-case-studies) shows you how to use our Docker images to get started with calling right away.

I know I've linked to a lot of docs, but there's a lot of context around each question you asked, so it would probably be easiest to go through those first. If you want more detail on a specific question, please feel free to follow-up!
